class Connect4Exception(Exception):
    pass
